README.md

```diff
@@ -39,6 +51,15 @@ sudo pacman -Syu && sudo pacman -S sdl2 libsdl2-devel sdl2_mixer sdl2-devel base
 
 ## Usage
 
+### FreeRTOS configuration
```

Comment:
Please move this section after `### Visual Studio Code` as setting up VSCode is the first step.